### PR TITLE
Create Moyo Catalog

### PR DESCRIPTION
**Pull request by mishakoshilka**
- [DEMO LINK](https://mishakoshilka.github.io/layout_catalog/)
- [TEST REPORT LINK](https://mishakoshilka.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/styles/blocks/card.scss [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(-n +1),
+.stars--2 .stars__star:nth-child(-n +2),
+.stars--3 .stars__star:nth-child(-n +3),
+.stars--4 .stars__star:nth-child(-n +4),
+.stars--5 .stars__star:nth-child(-n +5) {
+  background-image: url(../images/star-active.svg);
+}
```
Comment (anastasiiavorobiova) [inline src/styles/blocks/card.scss:95]: Consider using a separate file for stars block

File: src/styles/blocks/card.scss [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+.reviews__text {
```
Comment (anastasiiavorobiova) [inline src/styles/blocks/card.scss:97]:
```suggestion
&__text {
```

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Consider using SASS fetures in your code

File: src/styles/blocks/card.scss [hunk 3]
```diff
@@ -0,0 +1,150 @@
+:root {
+  --blue-accent: #00acdc;
+  --white: #fff;
+  --gray-color: #616070;
+  --dark-blue-color: #060b35;
+}
+
+.page {
```
Comment (vlad-demchuk) [inline src/styles/blocks/card.scss:8]: it looks like a separate bem block, you should create a separate file for each bem block

File: src/styles/blocks/card.scss [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+  font-family: "Roboto", sans-serif;
+}
+
+body {
```
Comment (vlad-demchuk) [inline src/styles/blocks/card.scss:12]: move to reset.css

File: src/styles/blocks/card.scss [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 198px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  transition: transform 0.3s;
+}
+
+.card__image {
```
Comment (vlad-demchuk) [inline src/styles/blocks/card.scss:27]: use nesting

File: src/styles/blocks/card.scss [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+
+  margin-right: 84px;
+  margin-left: 16px;
+  margin-bottom: 16px;
+}
+
+.reviews {
```
Comment (vlad-demchuk) [inline src/styles/blocks/card.scss:63]: no need to create a bem block for reviews

File: src/styles/blocks/card.scss [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+    justify-content: flex-end;
+
+    text-align: right;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+}
+
+.price {
```
Comment (vlad-demchuk) [inline src/styles/blocks/card.scss:87]: the same here

File: src/styles/blocks/card.scss [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+
+.price__cost {
+  color: var(--dark-blue-color);
+  text-align: right;
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+}
+
+.buy {
```
Comment (vlad-demchuk) [inline src/styles/blocks/card.scss:111]: the same here

File: src/styles/blocks/header.scss [hunk 9]
```diff
@@ -0,0 +1,67 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  margin-bottom: 50px;
+}
+
+.logo {
```
Comment (vlad-demchuk) [inline src/styles/blocks/header.scss:9]: the same here, make logo a bem element of header

File: src/styles/blocks/header.scss [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+  width: 40px;
+  height: 40px;
+}
+
+.logo__image {
+  width: 100%;
+  height: 100%;
+}
+
+.nav {
```
Comment (vlad-demchuk) [inline src/styles/blocks/header.scss:20]: move to a separate file

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]

File: src/styles/blocks/card.scss (hunk 6 above)
Comment (mishakoshilka) [reply src/styles/blocks/card.scss:63]: Anastasia Vorobiova said that it is necessary

File: src/styles/blocks/card.scss [hunk 11]
```diff
@@ -0,0 +1,142 @@
...
+    text-align: center;
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+  }
+}
+
+.card:hover {
```
Comment (anastasiiavorobiova) [inline src/styles/blocks/card.scss:125]:
```suggestion
.card {
  ...
  
  &:hover {
    ...
  }
```

Comment (anastasiiavorobiova) [reply src/styles/blocks/card.scss:125]: Consider using sass features for your code

File: src/styles/blocks/card.scss [hunk 12]
```diff
@@ -0,0 +1,142 @@
...
+  transform: scale(1.2);
+}
+
+.card:hover .card__product {
```
Comment (anastasiiavorobiova) [inline src/styles/blocks/card.scss:129]: The same here

File: src/styles/blocks/card.scss [hunk 13]
```diff
@@ -0,0 +1,142 @@
...
+  color: #34568b;
+  transition: color 0.3s;
+}
+
+.buy:hover {
```
Comment (anastasiiavorobiova) [inline src/styles/blocks/card.scss:134]: See my previous comment

File: src/styles/main.scss [hunk 14]
```diff
@@ -1,3 +1,6 @@
-body {
-  margin: 0;
-}
+@import "blocks/header";
+@import "blocks/card";
+@import "blocks/products-main";
+@import "blocks/stars";
+@import "blocks/reset";
+@import "blocks/nav";
```
Comment (anastasiiavorobiova) [inline src/styles/main.scss:6]: Consider using the right order for imports: 
1) For variables;
2) For basic styling, entire page styling
3) Blocks one by one from top to bottom

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Keep doing! Consider implementing your styles using scss features

**Review by anastasiiavorobiova — APPROVED** [review]
Awesome!

**Pull request closed without merging**